test/integration/goldens/logging/LoggingClient.java

```diff
@@ -613,6 +613,14 @@ public final ListLogEntriesPagedResponse listLogEntries(ListLogEntriesRequest re
    *
    * <pre>{@code
    * try (LoggingClient loggingClient = LoggingClient.create()) {
+   *   ListLogEntriesRequest request =
+   *       ListLogEntriesRequest.newBuilder()
+   *           .addAllResourceNames(new ArrayList<String>())
+   *           .setFilter("filter-1274492040")
+   *           .setOrderBy("orderBy-1207110587")
+   *           .setPageSize(883849137)
+   *           .setPageToken("pageToken873572522")
```

Comment:
Setting page token in the initial request seems harmful before you start paginated call there is no page token (i.e. page token makes sense only for pages 2+, but the first page does not have page token).

Reply:
FYI @miraleung